### PR TITLE
Updated note about execution_time_limit

### PR DESCRIPTION
**Pull request by dradulescu**
Inspired by this: https://github.com/renderedtext/improvement-proposals/issues/751

### CODE REVIEW

**Review by AleksandarCole — COMMENTED** [review]
@DamjanBecirovic can you please check this one and clarify If this docs update is accurate? 

I am confused about what was the solution for this: https://github.com/renderedtext/improvement-proposals/issues/751
After reading the ticket I'm under the impression that the goal of the IP was to exclude quota waiting time from the execution time limit?

File: docs/reference/pipeline-yaml-reference.md [hunk 1]
```diff
@@ -183,8 +183,10 @@ pipeline and second, with a `block` scope where you define the execution time
 limit for a single block. In the latter case you can use the `execution_time_limit`
 property in multiple blocks.
 
-Please note that the pipeline time limit is time limit for the sum of the
-running times of all the blocks of the pipeline.
+*Note*: The pipeline time limit is time limit for the sum of the
+running times of all the blocks of the pipeline, this also includes the time jobs
+are waiting for quota. Please keep this in mind when using the 
+`execution_time_limit` proberty.
```
[the remark this answers is not in the document]
Comment (dradulescu) [reply docs/reference/pipeline-yaml-reference.md:189]: How did I even bork it up so bad? This is meme tier stuff :see_no_evil:

**Comment by DamjanBecirovic** [comment]
@AleksandarCole the goal of IP was achieved by introducing the option to define `execution_time_limit` on the job level.
This was necessary because, for example, when time limit on pipeline level is considered, each block is seen as black-box that is either running or done so there is no way to know if any of the jobs within the block is waiting for the quota.
On the other hand, when time limits are defined and considered on a job level we can exclude the time job is waiting for quota.

@dradulescu the `Note` update is fine, but what needs to be introduced in docs is that there is now a third way for defining the `execution_time_limit` in pipeline YAML files. Here is an example:
``` yaml
version: v1.0
name: The execution_time_limit on a yob level
agent:
  machine:
    type: e1-standard-2
    os_image: ubuntu1804

blocks:
  - name: Tests
    task:
      jobs:

      - name: Lint
        commands:
          - checkout
          - make lint

      - name:  Tests
        execution_time_limit:
          minutes: 30
        commands:
          - checkout
          - make test
```

File: docs/reference/pipeline-yaml-reference.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dradulescu) [reply docs/reference/pipeline-yaml-reference.md]: Fixed in the latest push ;)

**Comment by dradulescu** [comment]
Thanks for the detailed response @DamjanBecirovic! Added the job scope and an example :crossed_fingers:

**Review by AleksandarCole — APPROVED** [review]

**Pull request merged**